### PR TITLE
Issue 1 create an entry - Daniel Cho

### PR DESCRIPTION
**Pull request by DanielC2022**
Kapture 2022-04-04 at 21 40 32
I realize resolution is a little poor, but I to select this conversion rate to keep it below 10MB

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
 /* exported data */
+
+var $photoInput = document.querySelector('.photo-input');
+var $image = document.querySelector('.image');
+var $form = document.querySelector('.form');
+
+$photoInput.addEventListener('input', function (event) {
+  var newInput = event.target.value;
+  var previewImg = $image.setAttribute('src', newInput);
+  return previewImg;
```
Comment (Zhiwutian) [inline js/main.js:11]: don't need to return anything here.

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+
+var newObjEntries = localStorage.getItem('js-local-storage');
+if (newObjEntries !== null) {
+  data.entries = JSON.parse(newObjEntries);
+}
+
+window.addEventListener('beforeunload', function (event) {
+  var dataEntries = JSON.stringify(data.entries);
+  localStorage.setItem('js-local-storage', dataEntries);
+});
```
Comment (Zhiwutian) [inline js/main.js:35]: this should be in data.js, and you should store the entire data object.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates needed!

File: js/main.js (hunk 1 above)
Comment (DanielC2022) [reply js/main.js]:
```
$photoInput.addEventListener('input', function (event) {
  var newInput = event.target.value;
  $image.setAttribute('src', newInput);
});
```

File: js/main.js (hunk 2 above)
Comment (DanielC2022) [reply js/main.js]:
```
var newObjEntries = localStorage.getItem('js-local-storage');
if (newObjEntries !== null) {
  data.entries = JSON.parse(newObjEntries);
}

window.addEventListener('beforeunload', function (event) {
  var dataObj = JSON.stringify(data);
  localStorage.setItem('js-local-storage', dataObj);
});
```

Comment (DanielC2022) [reply js/main.js]:
```

var newObject = localStorage.getItem('js-local-storage');
if (newObject !== null) {
data = JSON.parse(newObject);
}

window.addEventListener('beforeunload', function (event) {
var dataObj = JSON.stringify(data);
localStorage.setItem('js-local-storage', dataObj);
});

```

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**